src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSFunctionBinding.cs

```diff
@@ -433,7 +433,7 @@ internal static unsafe void ResolveOrRejectPromise(JSProxyContext targetContext,
             {
                 fixed (JSMarshalerArgument* ptr = arguments)
                 {
-                    Interop.Runtime.ResolveOrRejectPromise((nint)ptr);
+                    Interop.Runtime.ResolveOrRejectPromise((void*)ptr);
```

Comment:
```suggestion
                    Interop.Runtime.ResolveOrRejectPromise((nint)ptr);
```
Seems to have been missed (will require corresponding changes in `Interop.Runtime.NativeAOT.cs`).